tests/test_webviz_subsurface_testdata.py

```diff
@@ -0,0 +1,119 @@
+"""Testing loading the webviz subsurface testdata."""
+
+from __future__ import absolute_import
```

Comment:
Do we need this now when we are python3 only?

Reply:
No, it can go away.

Reply:
removed.